docs/toml.md

```diff
@@ -1780,6 +1797,8 @@ Træfik needs the following policy to read ECS information:
             "Sid": "Traefik ECS read access",
             "Effect": "Allow",
             "Action": [
+                "ecs:ListClusters",
```

Comment:
I don't see any calls to `ListClusters` or `DescribeClusters` in the ecs provider code.